components/calendar/index.css

```diff
@@ -205,29 +276,29 @@ governing permissions and limitations under the License.
   }
 
   &.is-today {
-    font-weight: var(--spectrum-calendar-day-today-text-font-weight);
+    font-weight: var(--mod-calendar-day-today-text-font-weight, var(--spectrum-calendar-day-today-text-font-weight));
   }
 
   &.is-range-selection {
     /* overrides to make the cells appear connected */
-    margin: var(--spectrum-calendar-day-padding) 0;
-    border-width: var(--spectrum-calendar-border-width-reset);
-    line-height: var(--spectrum-calendar-day-height);
-    border-radius: var(--spectrum-calendar-border-radius-reset);
+    margin: var(--mod-calendar-day-padding, var(--spectrum-calendar-day-padding)) 0;
+    border-width: var(--mod-calendar-border-width-reset, var(--spectrum-calendar-border-width-reset));
+    line-height: var(--mod-calendar-day-height, var(--spectrum-calendar-day-height));
+    border-radius: var(--mod-calendar-border-radius-reset, var(--spectrum-calendar-border-radius-reset));
 
     /* Todo: Calculate this */
```

Comment:
Can this comment be removed since you're calculating it?

Reply:
yeah the comments in this file are a little all over. I don't see a reason for this one to but here anymore though.